mods/ts/mod.yaml

```diff
@@ -61,17 +61,26 @@ Packages:
 	~ecache01.mix:CRC32
 
 Rules:
-	./mods/ts/rules/misc.yaml
 	./mods/ts/rules/ai.yaml
-	./mods/ts/rules/civilian.yaml
-	./mods/ts/rules/civilian-vehicles.yaml
+	./mods/ts/rules/misc.yaml
 	./mods/ts/rules/player.yaml
 	./mods/ts/rules/world.yaml
-	./mods/ts/rules/aircraft.yaml
 	./mods/ts/rules/defaults.yaml
-	./mods/ts/rules/infantry.yaml
-	./mods/ts/rules/structures.yaml
-	./mods/ts/rules/vehicles.yaml
+	./mods/ts/rules/aircraft.yaml
+	./mods/ts/rules/civilian-infantry.yaml
+	./mods/ts/rules/civilian-structures.yaml
+	./mods/ts/rules/civilian-vehicles.yaml
+	./mods/ts/rules/gdi-defenses.yaml
+	./mods/ts/rules/gdi-infantry.yaml
+	./mods/ts/rules/gdi-structures.yaml
+	./mods/ts/rules/gdi-vehicles.yaml
+	./mods/ts/rules/nod-defenses.yaml
+	./mods/ts/rules/nod-infantry.yaml
+	./mods/ts/rules/nod-structures.yaml
+	./mods/ts/rules/nod-vehicles.yaml
+	./mods/ts/rules/player-infantry.yaml
```

Comment:
How is this any different from `gdi-` and `nod-`? The name is also a bit misleading.